tests/device_tests/test_msg_cardano_sign_tx_slip39_basic.py

```diff
@@ -147,7 +150,16 @@ def input_flow():
         client.set_expected_responses(expected_responses)
         client.set_input_flow(input_flow)
         response = cardano.sign_tx(
-            client, inputs, outputs, fee, ttl, protocol_magic, network_id
+            client,
```

Comment:
at this point, I would consider passing this as keyword arguments

Reply:
Makes sense. Fixed.